packages/devops/scripts/deployment-vm-app/install.sh

```diff
@@ -1,10 +1,14 @@
-#!/bin/bash -ex
+#!/usr/bin/env bash
+set -ex
 
 HOME_DIR=/home/ubuntu
 TUPAIA_DIR=$HOME_DIR/tupaia
 LOGS_DIR=$HOME_DIR/logs
 
-SCRIPT_DIR=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+SCRIPT_DIR=$(
+    cd "$(dirname "${BASH_SOURCE[0]}")"
+    pwd -P
+)
```

Comment:
This is equivalent, right?

```suggestion
SCRIPT_DIR=$(realpath "${BASH_SOURCE[0]}")
```

Gonna leave it for a later PR, though. Keeping scope tight here to unblock deployments ASAP